docs/conf.py

```diff
@@ -86,6 +85,7 @@
 autodoc_member_order = "groupwise"
 autoclass_content = "both"
 autodoc_typehints_format = "short"
+autodoc_mock_imports = ["tree_sitter_languages", "llama-index"]
```

Comment:
After updating sphinx to 5.0.0, when building the docs, I was running into a different error:

```
Handler <function process_automodsumm_generation at 0x7fedb8d30430> for event 'builder-inited' threw an exception (exception: No module named 'gpt_engineer.tools.experimental.document_chunker')
make: *** [html] Error 2
```

I followed the advice [here](https://stackoverflow.com/questions/68398777/what-is-the-cause-of-this-sphinx-autodoc-mockfinder-error) and ensure that the two files were mocked when building the docs. This fixed the issue.